.github/workflows/test.yml

```diff
@@ -30,10 +30,10 @@ jobs:
           python setup.py install 
       - name: Test Linear Models
         run: |
-          python tests/test_linear_models/test_all.py
+          python tests/test_linear_models/test_linear_models.py
       - name: Test Pymilo Exceptions
         run: |
-          python tests/test_exceptions/test_all.py
+          python tests/test_exceptions/test_exceptions.py
```

Comment:
Don't you want to add Pytest to the `test.yml`? (like we do [here](https://github.com/sepandhaghighi/samila/blob/master/.github/workflows/test.yml#L40))

Reply:
you are right, sure.